src/babel.js

```diff
@@ -126,20 +131,47 @@ export default function ({types: t}) {
 
             if (el.name && el.name.name === 'style') {
               // we replace styles with the function call
-              const [id, css] = state.styles.shift()
+              const [id, css, loc] = state.styles.shift()
 
               const skipTransform = el.attributes.some(attr => (
                 attr.name.name === GLOBAL_ATTRIBUTE
               ))
 
+              // TODO: use plugin param option?
+              const useSourceMaps = process.env.NODE_ENV !== 'production'
```

Comment:
It might be a good idea to check the `sourceMaps` option of `babel` is enabled ? I don't know if it's possible though.